src/Endpoint.ts

```diff
@@ -33,7 +33,7 @@ export class Endpoint implements IEndpoint {
   base: string;
   availableFunctions: Array<string>;
   request: (params: Object) => Promise<any>;
-  search: (query: string, filters: object) => Function;
+  search: (query: string, filters: object) => Promise<Object>;
```

Comment:
Thanks for the PR - is `Promise<unknown>` more appropriate here?

Reply:
Yes it's a better solution